Monika After Story/game/script-anniversary.rpy

```diff
@@ -674,6 +675,34 @@ label anni_7:
     m 1ekbsa "Here's to another year together, and all the ones to follow."
     return "unlock"
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel='anni_8',
+            prompt="8 Years",
+            category=["anniversary"],
+            action=EV_ACT_QUEUE,
+            start_date=store.mas_anni.build_anni(years=8),
+            end_date=store.mas_anni.build_anni_end(years=8)
+        ),
+        skipCalendar=False
+    )
+
+label anni_8:
+    m 1hua "I'm sure you know what today is, [player]..."
+    m 3hublb "That's right! Happy anniversary, [mas_get_player_nickname()]!"
+    m 3ekblb "You know...when a relationship has lasted this long, sometimes couples start to worry that things might grow stale."
+    m 1lkbla "Eight years is a long time to learn someone's habits, and to build your lives around each other."
+    m 1lkc "Maybe...{w=0.5}you've worried about similar things, [player]?"
+    m 2ekb "I mean--{w=0.2}I don't want to assume!{w=0.3} But I was thinking about it earlier, and I wanted to tell you something."
+    m 2fkbsa "I wanted to tell you that it's been so exciting to watch you change."
+    m "In the time we've spent together, you've grown so much. You've gone through so many setbacks, but you survived every one."
+    m 2dkbsa "And yet,{w=0.2} with all of those changes you've been through..."
```

Comment:
I like what you did with the pooled tears. Let's use pooled tears here too, to start foreshadowing a little earlier?
```suggestion
    m 2dkbstpa "And yet,{w=0.2} with all of those changes you've been through..."
```